frontend/src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -0,0 +1,31 @@
+import NextAuth from "next-auth";
+import GithubProvider from "next-auth/providers/github";
+
+const handler = NextAuth({
+  providers: [
+    GithubProvider({
+      clientId: process.env.NEXT_PUBLIC_GITHUB_ClIENT_ID!,
+      clientSecret: process.env.NEXT_PUBLIC_GITHUB_CLIENT_SECRET!,
```

Comment:
Do we need the non-null assertion operator here?
```suggestion
      clientId: process.env.NEXT_PUBLIC_GITHUB_ClIENT_ID,
      clientSecret: process.env.NEXT_PUBLIC_GITHUB_CLIENT_SECRET,
```